src/main/java/com/faforever/client/chat/KittehChatService.java

```diff
@@ -428,17 +442,19 @@ public void connect() {
         .host(irc.getHost())
         .port(irc.getPort(), SecurityType.SECURE)
         .secureTrustManagerFactory(new TrustEveryoneFactory())
-        .password(getPassword())
         .then()
         .listeners()
         .input(this::onMessage)
         .output(this::onMessage)
         .then()
         .build();
 
+    nickServ = NickServ.builder(client).account(username).password(getPassword()).build();
```

Comment:
I'd love to see linebreaks on builder commands

Reply:
Got it boss